include/libopencm3/sam/3a/pmc.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2012 Gareth McMullin <gareth@blacksphere.co.nz>
+ * Copyright (C) 2015 Felix Held <felix-libopencm3@felixheld.de>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_PMC_H
+#define LIBOPENCM3_PMC_H
+
+#include <libopencm3/sam/memorymap.h>
+#include <libopencm3/sam/common/pmc_common_all.h>
+#include <libopencm3/sam/common/pmc_common_3a3s3x.h>
+#include <libopencm3/sam/common/pmc_common_3a3u3x.h>
```

Comment:
all of those files need to be included here. they need to be separate files because they contain defines for stuff that isn't present on all subfamilies and those subfamilies aren't subsets.